Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -386,6 +386,14 @@ public struct MissingAttributeArgument: ParserError {
   }
 }
 
+public struct MissingBothStringQuotesOfStringSegments: ParserError {
+  public let stringSegments: StringLiteralSegmentsSyntax
+
+  public var message: String {
+    return "expected \(nodesDescription([stringSegments], format: false)) to be surrounded by '\"'"
```

Comment:
Instead of escaping `"` then you could use `#` at the start and end of the string.
I think Rae strings reads a little easier

Reply:
Oh, that's great! Using a raw string seems more readable. I will make the change. Thank you 🙂